build-tests-samples/heft-storybook-react-tutorial-storykit/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "heft-storybook-react-tutorial-storykit",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Storybook build dependencies for heft-storybook-react-tutorial",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": ""
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.8",
```

Comment:
Does storybook require using Babel? That'd be unfortunate.

Reply:
@iclanton  The details are in #2833, but in a nutshell Storybook's "previewer app" is built using a completely distinct toolchain with Babel and its own Webpack config.  The purpose of my `-storykit` package pattern is to isolate that stuff so it doesn't pollute your Heft project.

We suspect that some of Storybook's dependencies can be eliminated, including Babel possibly -- for example if the "previewer app" is only used with evergreen browsers like Chrome.  That would be tackled as a separate PR, however, since it's somewhat unrelated to the Heft plugin.